application/dash_application/views/call_ready_wgs.py

```diff
@@ -178,8 +176,6 @@ def get_merged_wgs_data():
 initial["cutoff_coverage_tumour"] = 80
 cutoff_coverage_normal_label = "Coverage (Normal) minimum"
 initial["cutoff_coverage_normal"] = 30
-cutoff_tumor_purity_label = "Tumor Purity (%) minimum"
```

Comment:
You can also remove line 103-109 (code that loads the Purity data that is being removed) and the merge on 142.

Reply:
Done!